metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/StormControllerIntegrationTest.java

```diff
@@ -179,6 +181,8 @@ public void test() throws Exception {
     sensorParserConfig.setParserClassName("org.apache.metron.parsers.bro.BasicBroParser");
     sensorParserConfig.setSensorTopic("broTest");
     sensorParserConfigService.save(sensorParserConfig);
+    //we must wait for the config to find its way into the config.
+    Thread.sleep(500);
```

Comment:
Would it make sense to use assertEventually here instead of Thread.sleep?

Reply:
Yep, will change.